Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -28,10 +29,19 @@
 #include <AK/String.h>
 #include <AK/StringBuilder.h>
 #include <AK/Utf8View.h>
+#include <LibCrypto/BigInt/SignedBigInteger.h>
+// FIXME: This SHOULD NOT be required to build with Lagom
+// Compiler is complaining about: random_big_prime, LCM, GCD, ModularInverse
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
+#include <LibCrypto/NumberTheory/ModularFunctions.h>
+#pragma GCC diagnostic pop
```

Comment:
If someone can help me solving this in a nicer way that'd be great :D

Reply:
hmmm
could you try making them non-static?
if that fails, try marking them `inline` explicitly?